suite-common/wallet-core/src/send/sendFormReducer.ts

```diff
@@ -96,22 +104,24 @@ export const selectSendSerializedTx = (state: SendRootState) => state.wallet.sen
 export const selectSendSignedTx = (state: SendRootState) => state.wallet.send.signedTx;
 export const selectSendFormDrafts = (state: SendRootState) => state.wallet.send.drafts;
 
-export const selectSendFormDraftByAccountKey = (
+export const selectSendFormDraftByKey = (
```

Comment:
nit: 
```suggestion
export const selectSendFormDraftByDraftKey = (
```

Reply:
Feels redundant to me. It makes the name unnecessarily long.